packages/meditrak-app/app/Meditrak.js

```diff
@@ -44,7 +44,7 @@ class MeditrakContainer extends React.Component {
   renderBetaBanner() {
     return (
       <View style={localStyles.betaBanner} pointerEvents="none">
-        <Text style={localStyles.betaBannerText}>{betaBranch.toUpperCase()}</Text>
+        <Text style={localStyles.betaBannerText}>{betaBranch.toUpperCase() || centralApiUrl}</Text>
```

Comment:
```suggestion
        <Text style={localStyles.betaBannerText}>{centralApiUrl || betaBranch.toUpperCase()}</Text>
```
To consider, I'm not sure honestly. But part of me feels the banner is to help us know which server we're pointing to (so that we don't submit data un-intentionally).

Perhaps even:
```suggestion
        <Text style={localStyles.betaBannerText}>{`${betaBranch.toUpperCase()}${centralApiUrl ? ` -> ${centralApiUrl}` : ''}</Text>
```

All very optional btw, just some thoughts I had.

Reply:
Ah yeah you only really need to use one or the other. BETA_BRANCH is nice for non-technical people to get apk builds up, CENTRAL_API_URL is only something we would use on local. So the banner shows for both cases. Here's my local:
<img width="265" alt="Screen Shot 2023-07-04 at 1 30 29 pm" src="https://github.com/beyondessential/tupaia/assets/667652/4a2fdcd3-36de-447d-a77f-c4fb3d0a4a04">

Is it confusing having both? Maybe needs a little more comments?

Reply:
Oh I see. I think it's okay then, the comment you left in .env.example is clear enough